src/panels/lovelace/components/notifications/hui-notifications-button.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  html,
+  LitElement,
+  TemplateResult,
+  css,
+  CSSResult,
+  property,
+} from "lit-element";
+import "@polymer/paper-badge/paper-badge";
+import "@polymer/paper-icon-button/paper-icon-button";
+import { fireEvent } from "../../../../common/dom/fire_event";
+
+class HuiNotificationsButton extends LitElement {
+  @property() public notifications?: string[];
+  @property() public open?: boolean;
+
+  protected render(): TemplateResult | void {
+    return html`
+      <paper-icon-button
+        icon="hass:bell"
+        @click="${this._clicked}"
+      ></paper-icon-button>
+      ${this.notifications
+        ? html`
+            <paper-badge label="${this.notifications.length}"></paper-badge>
+          `
+        : ""}
+    `;
+  }
+
+  static get styles(): CSSResult[] {
+    return [
+      css`
+        :host {
+          position: relative;
+        }
+        paper-badge {
+          left: 23px !important;
```

Comment:
Why?

Reply:
It does but I thought it was too far right. Let me grab a screenshot

Reply:
![image](https://user-images.githubusercontent.com/1287159/52674947-c55de200-2eea-11e9-9438-2e573a23abf3.png)
vs
![image](https://user-images.githubusercontent.com/1287159/52674959-d0187700-2eea-11e9-8f03-71ae036a8401.png)

Reply:
Ah you're right, it does look better. 

How much do we need `paper-badge`? As we're trying to get rid of Polymer elements, it feels a bit weird to get one back in. Can't we update our existing dot to contain the number and look similar?

Reply:
I didn't think we were worrying about Polymer elements until we had a replacement; that being said, there's not much going on with paper-badge over the dot that was there, that is true. You're call.

Reply:
show me your CSS skillzzzzz

Reply:
prepare to be disappointed 😜